checker/src/test/java/org/checkerframework/checker/test/junit/NullnessOnlyAnnotatedForTest.java

```diff
@@ -0,0 +1,37 @@
+package org.checkerframework.checker.test.junit;
+
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * JUnit tests for the Nullness checker. This test suite tests -AonlyAnnotatedFor suppresses errors
+ * and warnings from the Nullness Checker for code outside the scope of @AnnotatedFor annotation.
+ */
+public class NullnessOnlyAnnotatedForTest extends CheckerFrameworkPerDirectoryTest {
+
+    /**
+     * Create a NullnessOnlyAnnotatedForTest.
+     *
+     * @param testFiles the files containing test code, which will be type-checked
+     */
+    public NullnessOnlyAnnotatedForTest(List<File> testFiles) {
+        super(
+                testFiles,
+                org.checkerframework.checker.nullness.NullnessChecker.class,
+                "nullness",
+                "-AonlyAnnotatedFor");
```

Comment:
Do you expect many tests that will use this flag?
If it's just this one or maybe another one, it's easier to just add a jtreg text under `checker/jtreg/nullness`.